alfred-telemetry-platform/src/main/resources/alfresco/module/alfred-telemetry-platform/alfresco-global.properties

```diff
@@ -35,6 +35,9 @@ alfred.telemetry.binder.enabled=true
 # Cache metrics disabled by default, because it outputs a lot of metrics at this point
 alfred.telemetry.binder.cache.enabled=false
 
+# Alfresco status
+alfred.telemetry.binder.alfresco-status.enabled=true
```

Comment:
Just a thought: `alfresco-` is maybe some duplication, since the module is to be installed in Alfresco.  
However I'm not convinced `alfred.telemetry.binder.status.enabled` is better.

Other opinions welcome.

Reply:
I tried to be consistent with alfresco-nodes. Not sure if that was reviewed by you.

Reply:
Apparently not :-D . It's just a thought, certainly no blocker.

Reply:
Btw, there are no reviewers yet.

Reply:
I'm not happy with `alfresco-nodes` either :-)

Reply:
I know, you are only happy with your own work.

Reply:
> Just a thought: `alfresco-` is maybe some duplication, since the module is to be installed in Alfresco.
> However I'm not convinced `alfred.telemetry.binder.status.enabled` is better.
> 
> Other opinions welcome.

One justification for using alfresco- here is the fact that these are alfresco metrics, as opposed to jvm, process, os. To me having alfresco in the metric name makes sense.

Reply:
Good point.